tests/php/Security/MemberAuthenticator/CookieAuthenticationHandlerTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace SilverStripe\Security\Tests\MemberAuthenticator;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\MemberAuthenticator\CookieAuthenticationHandler;
+
+class CookieAuthenticationHandlerTest extends SapphireTest
+{
+
+    protected $usesDatabase = true;
+
+    public function testLoginOnlySetsTokenWithPersistent()
+    {
+        $member = new Member(['Email' => 'test@example.com']);
+        $member->write();
+
+        $auth = new CookieAuthenticationHandler();
+        $auth->setTokenCookieName('testTokenCookie');
+
+        $headers = $this->runWithHeaderIsolation(function () use ($auth, $member) {
+            $auth->logIn($member, $persistent = true);
+        });
+        $cookieHeader = $this->getCookieHeader($headers);
```

Comment:
This method can return null, which won't work in the assertion below. If you change it to return an empty string instead of null then the assertion passes.

Reply:
Same here, not applicable to this PR